connector/src/main/scala/com/microsoft/kusto/spark/utils/KustoDataSourceUtils.scala

```diff
@@ -149,9 +153,9 @@ object KustoDataSourceUtils {
     if (!applicationId.isEmpty) {
       // Application authentication
       if(!applicationKey.isEmpty){
```

Comment:
Wouldn't these be clearer as IsNotEmpty or whatever the scala syntax is?

Reply:
If you think its clearer ill chnage